openshift/templates/email-verification-service/email-verification-service-deploy.yaml

```diff
@@ -4,15 +4,61 @@ apiVersion: v1
 metadata:
   name: ${NAME}
 objects:
-
+- kind: ExternalNetwork
+  apiVersion: security.devops.gov.bc.ca/v1alpha1
+  metadata:
+    name: ${NAME}${SUFFIX}
+    network: ${NAME}${SUFFIX}
+    labels:
+      name: ${NAME}${SUFFIX}
+      network: ${NAME}${SUFFIX}
+      app: ${APP_NAME}${SUFFIX}
+      app-group: ${APP_GROUP}
+      env: ${TAG_NAME}
+  spec:
+    description: |
+      Define the network parameters for accessing smtp server
+    entries:
+      - 0.0.0.0/0
+    servicePorts:
+      - tcp/25
+- kind: NetworkSecurityPolicy
+  apiVersion: security.devops.gov.bc.ca/v1alpha1
+  metadata:
+    name: ${NAME}
+    labels:
+      name: ${NAME}
+      env: ${TAG_NAME}
+      app: ${APP_NAME}${SUFFIX}
+      app-group: ${APP_GROUP}
+      env: ${TAG_NAME}
+  spec:
+    description: Allow agent to access the database and agent
+    source:
+      - - $namespace=${NAMESPACE_NAME}-${TAG_NAME}
+        - role=${ROLE}
+        - app=${APP_NAME}${SUFFIX}
+        - env=${TAG_NAME}
+    destination:
+      - - ext:name=${NAME}${SUFFIX}
+      - - role=db
+        - app=${APP_NAME}${SUFFIX}
+        - env=${TAG_NAME}
+        - $namespace=${NAMESPACE_NAME}-${TAG_NAME}
+      - - role=agent
+        - app=${APP_NAME}${SUFFIX}
+        - env=${TAG_NAME}
+        - $namespace=${NAMESPACE_NAME}-${TAG_NAME}
 - kind: Service
   apiVersion: v1
   metadata:
     name: ${NAME}
     labels:
-      name: ${NAME}
-      app: ${APP_GROUP}
+      app: ${APP_NAME}${SUFFIX}
+      name: ${NAME}${SUFFIX}
       app-group: ${APP_GROUP}
+      role: ${ROLE}
+      env: ${TAG_NAME}
   spec:
     ports:
     - name: ${NAME}
```

Comment:
Example of where not applying the ${SUFFIX} consistently could cause connectivity issues.

Reply:
added suffix to all NAME and APP_NAME